module/VuFind/src/VuFind/Controller/MyResearchController.php

```diff
@@ -909,8 +909,22 @@ public function mylistAction()
             };
 
             $results = $runner->run($request, 'Favorites', $setupCallback);
+            $listTags = [];
+
+            if ($this->listTagsEnabled()) {
+                $list = $results->getListObject();
+                $user = $this->getUser();
```

Comment:
Does the `$user` check here mean that tags are not visible on public lists unless the owner is logged in? Is that intentional? Is there a reason not to use `$list->getListTags()` here instead?

Reply:
This was not intentional. Edited to use `$list->getListTags` so that public list tags are displayed also for non-owners.